src/main/java/codeu/model/data/College.java

```diff
@@ -0,0 +1,15 @@
+public enum College {
+        BU("Boston University "),
+        Rice("Rice University"),
```

Comment:
Usually enums are capitals (BU, RICE, UIC), I'd change that to be consistent. 
There is a space, I'm not sure if you that's intentional or just a typo? BU("Boston University ").